pkg/cluster/config.go

```diff
@@ -20,6 +22,7 @@ type Config struct {
 	AWSIAMConfigs            map[string]*anywherev1.AWSIamConfig
 	GitOpsConfig             *anywherev1.GitOpsConfig
 	FluxConfig               *anywherev1.FluxConfig
+	SnowCredentialsSecret    *v1.Secret
```

Comment:
- [ ] Add a client config builder for this member, so we read from the cluster when building the `cluster.Config` in the controller